src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -40,35 +41,59 @@ const AccordionPanel = forwardRef(
       [active, theme.accordion.icons],
     );
 
+    // backward compatibility to the deprecated theme.accordion.hover.color
+    const headingColor =
+      theme.accordion.hover &&
+      theme.accordion.hover.color !== { dark: 'light-4', light: 'dark-3' }
+        ? theme.accordion.hover.color
+        : theme.accordion.hover.heading.color;
+
+    const { border: contentBorder } = theme.accordion;
+    const { border } = theme.accordion.panel;
+
+    let abutMargin;
+    if (border)
+      abutMargin = {
```

Comment:
Can we add a comment explaining what this is here for?

Reply:
Done, thanks.